src/utils/common-expression-helpers.ts

```diff
@@ -280,6 +280,17 @@ export class CommonExpressionHelpers {
     return daySinceLastObs === '' ? '0' : daySinceLastObs;
   };
 
+  generateInfantPTrackerId = (fieldId: string, motherPtrackerId: string): string | undefined => {
```

Comment:
Aren't these supported from individual esms? This looks specific to an implementation